common/src/models/models.h

```diff
@@ -23,10 +23,9 @@ extern "C" {
 
 // For integration into menu system
 void models_menu();
-void no_menu();
```

Comment:
Instead of reverting and removing this can we just change it to `no_models_menu`. And then in the embench I can change it from `no_menu` to `no_embench_menu`.

Reply:
Because removing this change technically handles this issue of the `no_menu` conflict during the build, but leaves the bug still of `no_menu` prototype missing if you include no models in the defines of the project Makefile.